### PR TITLE
refactor(main): initialize main project file

### PR DESCRIPTION
**Pull request by kunj-gandhi889**
Initialized Poetry environment.
Added all poetry configuration files and dependency required in this project.

To do: 

1. Add all fixed variables in constants package, have seperate files for different categories of constants.
2. Define all utlitly functions inside utils package.
3. Define a pipeline (class) in myapp package that contains all application logic.
4. main.py is entry point of application which will trigger the pipeline.

### CODE REVIEW

**Comment by jineshpatel2002** [comment]
Okay, Looks good to go

File: myapp/__init__.py
Comment (jineshpatel2002) [inline myapp/__init__.py:1]: I feel wrong commit message for this directory

Comment (kamleshshah15) [reply myapp/__init__.py:1]: I feel the package name is not appropriate.

**Review by kamleshshah15 — CHANGES_REQUESTED** [review]
This package name can be cli_app, file_processor_app or any other appropriate name

**Review by th-rutvi — APPROVED** [review]

**Review by kamleshshah15 — APPROVED** [review]

**Review by jineshpatel2002 — APPROVED** [review]

**Review by jineshpatel2002 — APPROVED** [review]

**Pull request merged**